src/cart/SConscript

```diff
@@ -71,11 +84,16 @@ def scons():
     env.AppendUnique(CPPPATH=[Dir('.').srcnode()])
     env.AppendUnique(CPPPATH=[Dir('include').srcnode()])
     env.AppendUnique(CPPPATH=[Dir('../gurt').srcnode()])
+    env.AppendUnique(CPPPATH=['#/src/include/cart'])
```

Comment:
This should be covered by line 84

Reply:
9f82c7570df9d37d821489ecca296e5e47726ad4

Reply:
On second thought, removing `-Isrc/include/cart` results in a missing `crt_utils.h` file:

```
$ gcc -o build/release/gcc/src/cart/crt_utils.os
  ...
  -Isrc/include/daos
  -Isrc/include
  -Ibuild/release/gcc/src/include
  -Isrc/cart/test/utest
  -Isrc/cart
  -Isrc/cart/include
  -Isrc/gurt
  -Iinstall/prereq/release/mercury/include
  -Iinstall/prereq/release/mercury/include/na
  -Iinstall/prereq/release/isal/include
  -Iinstall/prereq/release/isal_crypto/include
  -Iinstall/prereq/release/pmdk/include
  -Ibuild/release/gcc/src/tests/ftest
  /cart/utest
  src/cart/crt_utils.c

src/cart/crt_utils.c:14:23: fatal error: crt_utils.h: No such file or directory
#include "crt_utils.h"
```

Reply:
ah, the problem is that it should probably be in this directory rather than src/include/cart.  Headers in src/include/cart don't have crt_ prefix.  They are included in files using include <cart/api.h>.

But this is an internal header file so it should probably just go in this directory like crt_rpc.h

Reply:
10f7edf02303b4daae9bae088b84ac9b13004a86 https://github.com/daos-stack/daos/pull/5309/commits/cc94c8dfeaa1e3e2acebdd084dfd2895b96cec71